### PR TITLE
React Calculator: Milestone 2

### PR DESCRIPTION
**Pull request by jamezjaz**
**This is a real-world standard calculator built with React, a JavaScript library for building user interfaces.**
_On this branch, I implemented the following:_
- Prepared the directory structure.
- Implemented the App component and rendered Display, ButtonPanel, and Button components.
- Implemented the Display component which accepted the result of the calculation as an incoming prop, verified it with PropTypes, and set the default prop to 0.
- Implemented the Button component, it accepted a button name as a prop which is a string.
- Implemented the ButtonPanel component which renders the calculator panel with all buttons.

### CODE REVIEW

**Comment by sumancrest0001** [comment]
### Status: changes required :gear:

**Hello @jamezjaz,**
You have done good work and most of the requirements of this milestone are implemented. But there are some minor issues that need to be addressed before moving to the next milestone. Please consider the following points.

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+function App() {
+  return (
+    <>
+      <div className="App">
+        <header className="App-header">
+          <h3>REACT CALCULATOR</h3>
+        </header>
+        <Display result="5 * 5" />
+        <ButtonPanel />
+        <Button name="Calculator" />
```
Comment (sumancrest0001) [inline src/components/App.js]: **[Required]** Button should be imported and rendered in the ButtonPanel component, not in the App component. [Reference](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/addToComponent.pngl)

**[Required]** Commit messages should be capitalized. [Reference](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)

File: src/components/ButtonPanel.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    <button type="button">6</button>
+    <button type="button">-</button>
+    <button type="button">1</button>
+    <button type="button">2</button>
+    <button type="button">3</button>
+    <button type="button">+</button>
+    <button type="button" className="span-two">0</button>
+    <button type="button">.</button>
+    <button type="button">=</button>
+  </div>
```
Comment (sumancrest0001) [inline src/components/ButtonPanel.js]: **[Required]** Buttons component should be grouped as instructed in the requirements.[Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/buttonPanel/buttons.png)

For example, 1  2, 3, and +  should be grouped in a div.

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
**[Optional]** We can add all the CSS files in a directory. It will help in managing files and accessing them. But it is totally optional and you can do it in the future milestones also.

**Please make the necessary changes and submit for another code review. If you have any queries regarding the code review, please message me on Slack.**

**Suman Shrestha**

**Comment by jamezjaz** [comment]
Thank you very much, Suman.
I’ll reach out to you on slack now.

**Review by sumancrest0001 — APPROVED** [review]
### Status: Approved :+1: :1st_place_medal: 

**Congratulations @jamezjaz :clap: :clap:** 

Thank you for making the requested changes. Now, you can start working on the next milestone. 

**All the best with the next milestone
Happy coding!!! :slightly_smiling_face:**

**Pull request merged**